mayastor/src/core/env.rs

```diff
@@ -171,6 +172,7 @@ impl Default for MayastorCliArgs {
             core_list: None,
             bdev_io_ctx_pool_size: 65535,
             nvme_ctl_io_ctx_pool_size: 65535,
+            registration_endpoint: Uri::try_from("https://core:50051").unwrap(),
```

Comment:
If we provide a default value in the struct [here](https://github.com/openebs/mayastor/pull/1117/files#diff-b8c78c17c40a6080a4400f00a83ea2323dc8b4449513cd9a4f8a018099e7201bR85), should we hardcode the address here again?

Reply:
Yes, we need to pass again.